pkg/randstr/randstr_test.go

```diff
@@ -0,0 +1,71 @@
+package randstr_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ansible/receptor/pkg/randstr"
+)
+
+func TestRandStrLength(t *testing.T) {
+	randStringTestCases := []struct {
+		name           string
+		inputLength    int
+		expectedLength int
+	}{
+		{
+			name:           "length of 100",
+			inputLength:    100,
+			expectedLength: 100,
+		},
+		{
+			name:           "length of 0",
+			inputLength:    0,
+			expectedLength: 0,
+		},
+		{
+			name:           "length of -1",
+			inputLength:    -1,
+			expectedLength: 0,
+		},
+	}
+
+	t.Run("Test the random string length", func(t *testing.T) {
```

Comment:
Please move this `t.Run` inside the for loop and use the test case name to create 3 distinct tests. `t.Run(testCase.name, func(t *testing.T) {`

Reply:
ooops :smile: